exercises/concept/poetry-club-door-policy/.meta/exemplar.js

```diff
@@ -27,7 +27,7 @@
  * @returns {string}
  */
 export function frontDoorResponse(blurb) {
-  return blurb[0];
+  return blurb.trim()[0];
```

Comment:
Oh no, I just realized the instructions only say that backdoor response trims at the end. This line does not fit the instructions. It needs to stay as it was. Instead the "with spaces" test should only have additional spaces at end. Sorry for getting you on the wrong track there with my "with spaces" example.